sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -681,65 +681,75 @@ private void initialize() {
     private void processMinidumpFiles() {
 
         /* Convert minidump files to App Center crash files. */
-        for (File logFile : ErrorLogHelper.getNewMinidumpFiles()) {
-
-            /* Create missing files from the native crash that we detected. */
-            AppCenterLog.debug(LOG_TAG, "Process pending minidump file: " + logFile);
-            long minidumpDate = logFile.lastModified();
-            File dest = new File(ErrorLogHelper.getPendingMinidumpDirectory(), logFile.getName());
-            Exception modelException = new Exception();
-            modelException.setType("minidump");
-            modelException.setWrapperSdkName(WRAPPER_SDK_NAME_NDK);
-            modelException.setMinidumpFilePath(dest.getPath());
-            ManagedErrorLog errorLog = new ManagedErrorLog();
-            errorLog.setException(modelException);
-            errorLog.setTimestamp(new Date(minidumpDate));
-            errorLog.setFatal(true);
-            errorLog.setId(UUID.randomUUID());
-
-            /* Lookup app launch timestamp in session history. */
-            SessionContext.SessionInfo session = SessionContext.getInstance().getSessionAt(minidumpDate);
-            if (session != null && session.getAppLaunchTimestamp() <= minidumpDate) {
-                errorLog.setAppLaunchTimestamp(new Date(session.getAppLaunchTimestamp()));
-            } else {
+        for (File logFolder : ErrorLogHelper.getNewMinidumpFiles()) {
+            File[] files = logFolder.listFiles();
```

Comment:
Problem is if we update the SDK with pending crash in the previous directory, it will not be a directory but a simple file.

We need to check if directory and if it's a just a minidump file, process it with device object fallback like before.